docs: PCP-6629 + DOC-2740: MAAS SSH + Tags (#10479)
spectrocloud/librarium: docs/docs-content/clusters/data-center/maas/create-manage-maas-clusters.md

    | --------------------------------- | -------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
    | **Domains**                       | Register a DNS record in the selected domain for the deployed cluster. The DNS record links to the IP addresses of the control plane nodes.                                                                                                                                                                                                                                                                    |

>     | **SSH Keys (Optional)**    | Inject public Secure Shell (SSH) keys into `~/.ssh/authorized_keys` on each control plane and worker node, including LXD VMs. The list is populated using the keys in **Tenant Settings** or **Project Settings** > **Security** > **SSH Keys**. Select **Add Item** to add additional keys. Refer to [SSH Keys](/clusters/cluster-management/ssh/ssh-keys/) for additional information. |

Author: ```suggestion
    | **SSH Keys (Optional)**    | Inject public Secure Shell (SSH) keys into `~/.ssh/authorized_keys` on each control plane and worker node, including LXD VMs. The list is populated using the keys in **Tenant Settings** or **Project Settings** > **Security** > **SSH Keys**. Select **Add Item** to [add additional keys](/clusters/cluster-management/ssh/ssh-keys/). |
```

>     | **SSH Keys (Optional)**           | Inject public Secure Shell (SSH) keys into `~/.ssh/authorized_keys` on each cluster node. The list is populated using the keys in **Tenant Settings** or **Project Settings** > **Security** > **SSH Keys**. Select **Add Item** to add additional keys.                                                                                                                                                       |

Author: ```suggestion
    | **SSH Keys (Optional)**    | Inject public Secure Shell (SSH) keys into `~/.ssh/authorized_keys` on each control plane and worker node, including LXD VMs. The list is populated using the keys in **Tenant Settings** or **Project Settings** > **Security** > **SSH Keys**. Select **Add Item** to add additional keys. Refer to [SSH Keys](/clusters/cluster-management/ssh/ssh-keys/) for additional information. |
```

    | **NTP Servers (Optional)**        | Specify Network Time Protocol (NTP) servers for the cluster nodes. The servers you provide override the machine image defaults. We recommend specifying at least one NTP server to prevent time drift issues.                                                                                                                                                                                                  |

>     | **Host LXD-Based Control Planes** | Activate to use this cluster as an LXD-based control plane (hypervisor) for running control plane components as LXD VMs. To create a workload cluster that will leverage MAAS LXD or use an existing host LXD-based control plane, leave this option disabled. Refer to [Create and Manage MAAS Clusters Using LXD VMs](create-manage-maas-lxd-clusters.md) for more information on leveraging LXD in Palette. |

Author: ```suggestion
    | **Host LXD-Based Control Planes** | Activate to use this cluster as an LXD-based control plane (hypervisor) for running control plane components as LXD VMs. To create a workload cluster that leverages MAAS LXD or use an existing host LXD-based control plane, leave this option disabled. Refer to [Create and Manage MAAS Clusters Using LXD VMs](create-manage-maas-lxd-clusters.md) for more information on leveraging LXD in Palette. |
```

    | **Minimum CPU**        | Only select machines from the **Resource pool** that have at least the specified amount of CPU.                                                                                                                                                                                                                                                               |
    | **Minimum Memory**     | Only select machines from the **Resource pool** that have at least the specified amount of memory.                                                                                                                                                                                                                                                            |
    | **Availability zones** | The MAAS zones in which to deploy nodes. Palette distributes nodes across the selected zones for high availability.                                                                                                                                                                                                                                           |

>     | **Tags**               | Controls which MAAS machines to use for the node pool based on [MAAS tags](https://canonical.com/maas/docs/about-machine-groups#p-22953-tags). If you specify multiple tags, Palette uses AND logic and only selects machines that have all of the specified tags. <br /> <br /> **WARNING:** Modifying tags on an existing node pool triggers a node repave. |

Author: ```suggestion
    | **Tags**               | Controls which MAAS machines to use for the node pool based on [MAAS tags](https://canonical.com/maas/docs/about-machine-groups#p-22953-tags). If you specify multiple tags, Palette only selects machines that have _all_ of the specified tags. <br /> <br /> **WARNING:** Modifying tags on an existing node pool triggers a node repave. |
```
